File: src/gameEntities/Pickupable.java
package gameEntities;

import worldModel.utilities.Pair;
import worldModel.utilities.enums.Entities;

/**
 * Class defining the pickupable entities, those that the player can pick up
 *
 */
public class Pickupable extends GameEntity {

	public Pickupable(final Pair<Integer, Integer> position, final Entities typeEnt) {
		super(position, typeEnt);
	}

}

File: Pirazzoli_Federico/GameEntity/Pickupable.cs
using System;

using design.utilities;
using design.enums;

namespace gameEntities
{
    public class Pickupable : GameEntity
    {
        public Pickupable(Pair<int, int> position, Entities ent)
            : base(position, ent)
        {

        }
    }
}
